docs/svelte.config.js

```diff
@@ -33,16 +33,6 @@ const config = {
     vite: {
       build: {
         rollupOptions: {
-          external: [
-            '@web3-react/core',
-            '@web3-react/eip1193',
-            '@web3-react/metamask',
-            '@web3-react/network',
-            '@web3-react/walletconnect',
-            '@web3-react/types',
-            '@web3-react/url',
-            '@web3-onboard/*'
```

Comment:
I was wrong, we still want the `web3-onboard/*` within the external list. My bad!

Reply:
no prob- readded!